src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonCellModel.swift

```diff
@@ -64,5 +66,6 @@ class HealthCertifiedPersonCellModel {
 
 	let validityStateIcon: UIImage?
 	let validityStateTitle: String?
-
+	
+	let accessoryIcon: UIImage
```

Comment:
I would say it's not really necessary to have this in the view model as it always stays the same image. Doesn't need to be configured depending on the certificate for example. (The same could be said about the `title` property as well, doesn't need to be configured or tested either.)